bin/zssServer.sh

```diff
@@ -13,6 +13,14 @@ export _BPXK_AUTOCVT=ON
 COMPONENT_HOME="${ZWE_zowe_runtimeDirectory}/components/zss"
 ZSS_SCRIPT_DIR="${COMPONENT_HOME}/bin"
 
+# this is to resolve LIBPATH current working dir issue. If LIBPATH is set manually by overwriting all existing values then cwd doesn't get resolved
+if [ -n "$LIBPATH" ]
+then
+  if [ "$(echo "${LIBPATH}" | grep ":.:")" != "${LIBPATH}" ] && [ "$(echo "${LIBPATH}" | cut -c1-2)" != ".:" ] && [ "$(echo "${LIBPATH}" | tail -c 3)" != ":." ] && [ "$(echo "${LIBPATH}" | grep "${ZSS_SCRIPT_DIR}")" != "${LIBPATH}" ];
```

Comment:
```sh
[ "$(echo "${LIBPATH}" | grep "${ZSS_SCRIPT_DIR}")" != "${LIBPATH}" ]
```
Wouldn't this also be true if `ZSS_SCRIPT_DIR` is a substring of a longer path? I.e. this would be a false positive.

Reply:
@ifakhrutdinov addition of ZSS_SCRIPT_DIR without if condition doesn't cause any issue. As it is run by script it doesn't get affected by restarts.